property.ts

```diff
@@ -29,6 +29,7 @@ import {
  * @param seed The seed for reproducible property-based tests.
  * @param path The path for reproducible property-based tests.
  * @param runs The number of test runs.
+ * @param endOnFailure Whether to stop fuzzing after the first failure.
```

Comment:
Is this precise? I was thinking of something along the lines of 'do not attempt to shrink' after the first failure, since shrinking kicks in after a failure occurs.

```suggestion
 * @param endOnFailure Stop execution after the first failure and prevent further shrinking.
```